ddp-workspace/projects/ddp-pancan/src/assets/i18n/es.json

```diff
@@ -369,7 +369,13 @@
             "ReturnButton": "REGRESAR A PÁGINA PRINCIPAL"
         },
         "Dashboard": {
-            "Title": "Tablero de Progreso"
+            "Title": "Tablero de Progreso",
+            "ParticipantsTitle": "[ES]Participant Dashboard",
+            "AddParticipant": "[ES]Add Participant",
+            "ShowPanel": "[ES]Show",
+            "HidePanel": "[ES]Hide",
+            "UserLabel": "Usted",
+            "ChildLabel": "[ES]Your child"
```

Comment:
#optional
`"Su hijo(a)" ` 
(picked it up from `ddp-study-builder` email template in Spanish)

Reply:
done!